### PR TITLE
parser: add CreateViewPriv privilege

### PR DESCRIPTION
**Pull request by AndrewDi**
Part of View feature implement, add `CreateViewPriv` privilege to parser and related const.

### CODE REVIEW

File: mysql/const.go [hunk 1]
```diff
@@ -218,6 +218,8 @@ const (
 	ExecutePriv
 	// IndexPriv is the privilege to create/drop index.
 	IndexPriv
+	// CreateViewPriv is the privilege to create view.
+	CreateViewPriv
```
Comment (AndrewDi) [inline mysql/const.go]: @XuHuaiyu @winkyao Could I put `CreateViewPriv` here like this?

**Comment by AndrewDi** [comment]
@lonng Could you use PR https://github.com/pingcap/tidb/pull/9153 to run parser test?

**Comment by lonng** [comment]
Do you mean that https://github.com/pingcap/tidb/pull/9153 uses this PR as the dependency? If it does, you could change `go.mod` of `tidb#9153`. See: `README.md`.

**Comment by XuHuaiyu** [comment]
PTAL @tiancaiamao

**Comment by tiancaiamao** [comment]
LGTM

**Comment by tiancaiamao** [comment]
Did you forget to call `make parser` before commit? @AndrewDi

**Comment by AndrewDi** [comment]
Emm...you just `Merge branch 'master' into view/parser_create_view_priv`, that will make parser.go mixed up...

**Comment by AndrewDi** [comment]
> Oh, I see... Would you like update this branch please?
Done

**Pull request merged**